doc/api.yml

```diff
@@ -1400,6 +1640,16 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/StudentExamGrade'
+    CreatePointing:
+      type: object
+      properties:
+        pointing_type:
+          $ref: '#/components/schemas/PointingType'
+        place:
+         type: string
+         enum:
```

Comment:
Ah ! Here it is. 

Set it as Component the use it every where !

Reply:
already moved to a Component